extra/cloudformation/default-unauth-resources-template.yaml

```diff
@@ -647,7 +647,7 @@ Resources:
     Properties:
       FunctionName: AmazonLocationDemoIotEndpointProvider
       Handler: index.handler
-      Runtime: nodejs16.x
+      Runtime: nodejs20.x
```

Comment:
Have you tested out this change by verifying the lambda still executes correctly? I was under the impression that nodejs18+ in lambda doesn't support aws-sdk v2 (the "var aws = require("aws-sdk");" line below), and would throw package not found errors.

Reply:
Hey @mbalfour-amzn updated the lambdas accordingly, and tested them as well. Should be good now!